.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Bug Report 
+about: Report an issue
+---
+## Prerequisites
+<!--- Go through the items below before logging an issue -->
+- [ ] I have searched in this repository's issues to see if it has already been reported.
+- [ ] This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
+- [ ] I have updated to the latest released version of the SDK and the issue still persists.
+
+## Bug Summary 
+
+<!---Please provide a summary of the bug you are reporting-->
+
+
+## Steps to Reproduce
+
+1.
+2.
+3.
+
+## Expected Behavior
+
+<!---What is the expected behavior?-->
+
+## Actual Behavior
+
+<!---What is the Actual behavior you see?-->
+
+## Platform and Version
+
+Which platform? 
+Which SDK version(s)?
+Which IDE version?
+
+## Sample Code that illustrates the problem
+<!---Please provide sample code below or provide a link with the sample code to help us investigating the bug-->
+
+```
+// sample code
+
+```
+## Logs taken while reproducing the problem
+<!---Please provide Trace Console Log below or provide a link with the log to help us looking into the issue-->

```

Comment:
looking -> look

Reply:
I also changed the "investigating" to "investigate" in the sample code section.